CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -121,6 +121,37 @@ protected function getTemplateUpdates() {
           ['name' => 'pledge_acknowledge', 'type' => 'html'],
         ],
       ],
+      [
+        'version' => '5.20.alpha1',
+        'upgrade_descriptor' => ts('Aditional email greetings and translation fixes'),
```

Comment:
@magnolia61 templates name should be the one without _html.tpl the upgrade tests failed because it couldn’t find the relevant template file